src/Component/Header/Header.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import MainNavbar from './MainNavbar';
+import CATEGORY from './categoryData';
+import './Header.scss';
+
+class Header extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      menu: ['LOGIN', 'JOIN', 'CART', 'ORDER'],
+      category: [
+        'HEADWEAR',
+        'BAG',
+        'CLOTHES',
+        'SHOES',
+        'ACC',
+        'COMMUNITY',
+        'LOOKBOOK',
+      ],
+      categoryList: [],
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      categoryList: CATEGORY,
+    });
+  }
```

Comment:
```suggestion
  componentDidMount() {
    fetch('/data/CategoryData.json')
      .then(res => res.json())
      .then(data => {
        this.setState({
          categoryList: data,
        });
      });
  }
```

이런식으로 변경하는 것이 나중에 백엔드랑 통신을 위해서도 훨씬 유용합니다!
이렇게 사용하려면 현재 구성중인 mockdata를
1. public 하위에 data 폴더를 만들고 하위에 CategoryData.json 파일을 만들어서 진행해주세요!
2. 나중 충돌을 생각해서 json 파일 이름을 HeaderCategory 이런식으로 변경해주세요~